invenio_app/ext.py

```diff
@@ -67,8 +67,10 @@ def init_app(self, app, **kwargs):
         # Enable PING view
         if app.config['APP_HEALTH_BLUEPRINT_ENABLED']:
             blueprint = Blueprint('invenio_app_ping', __name__)
+            limiter = self.limiter
 
             @blueprint.route('/ping')
+            @limiter.exempt
```

Comment:
I would add a test also to be sure this works, e.g. set `RATELIMIT_APPLICATION = '1/day'` and try to make multiple requests to the `/ping` view

Reply:
Yes I thought so but I didn't test it as in principle is like testing the decorator itself, so I assumed is tested. But yes I can add one!